csm/core/blogic/const.py

```diff
@@ -945,3 +945,8 @@
 REQUEST_CANCELLED       = 'RequestCancelled'
 SETUP_ERROR             = 'SetupError'
 HTTP_ERROR              = 'HttpError'
+
+# Session Backend factory
+DB                  = 'database'
```

Comment:
Please change all these values accordingly

Reply:
updated the const